lib/cubit/task_cubit.dart

```diff
@@ -19,18 +21,18 @@ class TaskCubit extends Cubit<TaskState> {
 
   Future<void> readTasks() async {
     TaskService service = const TaskService();
```

Comment:
Next time we meet I want to show you how we can avoid this line being written a bunch of times. Please remind me when we meet and please resolve this comment.

Reply:
I moved it above the functions and just reused `service`, is this what you meant?

Reply:
Close to ideal. This is a definite improvement. There is something you can do to make this file testable. I'll show you and explain.